lightning/src/ln/peer_handler.rs

```diff
@@ -953,7 +966,19 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 		peer: &mut Peer,
 		message: wire::Message<<<CMH as core::ops::Deref>::Target as wire::CustomMessageReader>::CustomMessage>
 	) -> Result<Option<wire::Message<<<CMH as core::ops::Deref>::Target as wire::CustomMessageReader>::CustomMessage>>, MessageHandlingError> {
-		log_trace!(self.logger, "Received message {:?} from {}", message, log_pubkey!(peer.their_node_id.unwrap()));
+		match message {
+			wire::Message::Ping(_) |
+			wire::Message::Pong(_) |
```

Comment:
Likewise?

Reply:
Yep, removed `Ping`/`Pong` and added `ChannelUpdate` with https://github.com/rust-bitcoin/rust-lightning/pull/1145/commits/753889b036d52eba9a95a57a610771d602cf8f74.